tests/components/irm_kmi/test_coordinator.py

```diff
@@ -0,0 +1,19 @@
+"""Tests for the coordinator of IRM KMI integration."""
+
+from datetime import timedelta
+from unittest.mock import MagicMock
+
+from homeassistant.components.irm_kmi.coordinator import IrmKmiCoordinator
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def test_jules_forgot_to_revert_update_interval_before_pushing(
```

Comment:
Can you give this a clearer name? We don't need unit tests to blame developers for their bugs. 😉